### PR TITLE
feat: Add IDMS DML support for MAP statements part 2 #704

### PR DESCRIPTION
**Pull request by schte01**
Signed-off-by: Teri Schnitt <teri.schnitt@broadcom.com>

### CODE REVIEW

File: server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CICSParser.g4 [hunk 1]
```diff
@@ -979,45 +979,73 @@ cics_value: LPARENCHAR ptr_value RPARENCHAR;
...
+   | TRANSACTION | TRUNCATED | TYPEDEF
+   | UPDATE | USER | UTF_8
    | VERSION | VIRTUAL
-   | WAIT | WITHIN
-   | XCTL
+   | WAIT | XCTL
    | YEAR | YYYYMMDD | YYYYDDD
    ;
 
+idms_only_words
```
Comment (temanbrcom) [inline server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CICSParser.g4:1014]: 👍

**Review by temanbrcom — APPROVED** [review]
Nicely done, I will update the coloring and token names for errors after the merge.

**Review by asatklichov — APPROVED** [review]
🔢

**Comment by asatklichov** [comment]
OOM happens

**Pull request merged**